3rdparty/7z/CMakeLists.txt

```diff
@@ -45,7 +45,15 @@ if(WIN32)
 		src/LzmaLib.c
 		src/7zCrc.c
 		src/MtDec.c)
+	
+	target_include_directories(3rdparty_7z
```

Comment:
Why is this in 3 lines? One should be plenty enough

Reply:
style was copied from elsewhere in the codebase, theres no style guide so i didnt think to change it